docs/products/forms.mdx

```diff
@@ -92,8 +112,12 @@ Set heights using classes like .form-control-lg and .form-control-sm.
 ## Radio
 <Playground>
 <div class="custom-control custom-radio">
-  <input type="radio" id="customRadio1" name="customRadio" class="custom-control-input"/>
-  <label class="custom-control-label" for="customRadio1">Label</label>
+  <input type="radio" id="customRadio1" name="customRadio" class="custom-control-input" checked/>
```

Comment:
@Pallavi-PH Checked `Checkbox`  color will be primary or green here ? many places seen input type checkbox checked have green theme.

Reply:
It will be green, as in the screenshot

Reply:
Ok ,Thanks